quic/s2n-quic-transport/src/endpoint/mod.rs

```diff
@@ -342,6 +342,7 @@ impl<Cfg: Config> Endpoint<Cfg> {
             self.connections.handshake_connections(),
             self.connections.len(),
             &remote_address,
+            timestamp.into_event(),
```

Comment:
So here I converted the timestamps and I am curious if this is the right thing to do. At this point I ran into a lot of problems with the types and found myself mucking around with the `s2n-quic-events` and was having problems with `generated.rs` which felt wrong...